azure/services/managedclusters/spec.go

```diff
@@ -122,6 +122,9 @@ type ManagedClusterSpec struct {
 
 	// OIDCIssuerProfile is the OIDC issuer profile of the Managed Cluster.
 	OIDCIssuerProfile *OIDCIssuerProfile
+
+	// This cannot be updated once the Managed Cluster has been created.
```

Comment:
```suggestion
	// DNSPrefix is ... This cannot be updated once the Managed Cluster has been created.
```
Start doc comment with name of field and what it does.